<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AUserInterfaceClass</class>
 <widget class="QWidget" name="AUserInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LLScanner</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>RGB Frame</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="LabelRGBFrame">
          <property name="minimumSize">
           <size>
            <width>256</width>
            <height>192</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>256</width>
            <height>192</height>
           </size>
          </property>
          <property name="text">
           <string>RGB Frame</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Laser Frame</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="LabelLaserFrame">
          <property name="minimumSize">
           <size>
            <width>256</width>
            <height>192</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>256</width>
            <height>192</height>
           </size>
          </property>
          <property name="text">
           <string>Laser Frame</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Calibration File System</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QLabel" name="SCPathText">
          <property name="text">
           <string>SC Path</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="CCPathText">
          <property name="text">
           <string>CC Path</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="SCUpdateButton">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="CCLoadButton">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>CC Load</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="SCLoadButton">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>SC Load</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="CCUpdateButton">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="TopLayout">
     <item>
      <widget class="QGroupBox" name="VisibleGroup">
       <property name="title">
        <string>Visible</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="VisibleDefault">
          <property name="text">
           <string>Default</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="VisibleHBoard">
          <property name="text">
           <string>HBoard</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="VisibleVBoard">
          <property name="text">
           <string>VBoard</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="VisibleLineLaser">
          <property name="text">
           <string>Line Laser</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="VisibleCursorPosition">
          <property name="text">
           <string>Cursor Position</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="UpdateGroup">
       <property name="title">
        <string>Update</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="UpdateHBoardBox">
          <property name="text">
           <string>Update H Board</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="UpdateVBoardBox">
          <property name="text">
           <string>Update V Board</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="UpdateLineLaserBox">
          <property name="text">
           <string>Update Line Laser</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="UpdateNoneBox">
          <property name="text">
           <string>Update None</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="InformationGroup">
       <property name="title">
        <string>Information</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="InfoHBoard">
          <property name="text">
           <string>H Board</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="InfoVBoard">
          <property name="text">
           <string>V Board</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="InfoLineLaser">
          <property name="text">
           <string>Line Laser</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SCSaveButton">
          <property name="maximumSize">
           <size>
            <width>170</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Save Calibration Data</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="SerialGroup">
       <property name="title">
        <string>Serial</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="SerialStateLabel">
          <property name="text">
           <string>Connection dosen't exist</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="ComSelectLayout">
          <item>
           <widget class="QSpinBox" name="SerialChannel">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ConnectButton">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="UnitStepButton">
          <property name="text">
           <string>Unit Step</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="BotLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Scan</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Hyper Parameter</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <widget class="QSpinBox" name="RThSpinBox">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <number>-255</number>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Window Size</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Odometry CC</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSlider" name="LimXYSlider">
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="LimXYText">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QSlider" name="MeaCCSlider">
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="value">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Limit Z</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_2">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Measurement CC</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="WSizeSpinBox">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>32</number>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSlider" name="LimZSlider">
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Limit XY</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="LimZText">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QLabel" name="MeaCCText">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>0.01</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLabel" name="OdoCCText">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>0.1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Red Threshold</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSlider" name="OdoCCSlider">
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="value">
              <number>10</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QCheckBox" name="ExportBox">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Export</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="ScanDataClearButton">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="ScanBox">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Scan</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="ScanProcessingButton">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Processing</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="InitializeButton">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Initialize</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AUserInterface.qrc"/>
 </resources>
 <connections/>
</ui>
